src/scss/radial-progress.scss

```diff
@@ -38,4 +38,7 @@
         font-size: .8em;
         font-weight: 500;
     }
+    &:hover:after {
```

Comment:
I think is better to use

```
.item__radial-options input:hover:after {
    content: "GiB";
}
``` in `advanced.scss` but I can't figure out how to do this

Reply:
it's cool 👍  :)

Reply:
Really? :) Thanks!
I was trying to add `hover:after` input box but without success